Cargo.toml

```diff
@@ -19,8 +19,8 @@ dirs = "4.0"
 dunce = "1.0"
 env_logger = "0.9"
 futures = "0.3"
-hippo-openapi = "0.5"
-hippo = { git = "https://github.com/deislabs/hippo-cli"}
+hippo-openapi = { git = "https://github.com/fermyon/hippo-openapi" }
+hippo = { path = "../hippo-cli"}
```

Comment:
leaving this in here for now to indicate how to test. Testing change to hippo-cli now.